tests/everest/test_detached.py

```diff
@@ -294,3 +297,39 @@ def test_generate_queue_options_no_config():
 def test_generate_queue_options_use_simulator_values(queue_options, expected_result):
     config = EverestConfig.with_defaults(**{"simulator": queue_options})
     assert get_server_queue_options(config) == expected_result
+
+
+@pytest.mark.timeout(5)  # Simulation might not finish
+@pytest.mark.integration_test
+@pytest.mark.xdist_group(name="starts_everest")
+async def test_starting_not_in_folder(tmp_path, monkeypatch):
+    async def server_running():
+        while True:
+            event = await driver.event_queue.get()
+            if isinstance(event, FinishedEvent) and event.iens == 0:
+                return event
+
+    os.makedirs(tmp_path / "new_folder")
+    monkeypatch.chdir(tmp_path / "new_folder")
+    everest_config = EverestConfig.with_defaults()
+    everest_config.dump("minimal_config.yml")
+    everest_config.config_path = Path("minimal_config.yml").absolute()
+    monkeypatch.chdir(tmp_path)
+    monkeypatch.setenv("PATH", f".:{os.environ['PATH']}")
```

Comment:
Sure this part is needed? (locally) I tried removing it and got the same fail/success behavior wrt the config path

Reply:
It is needed if we want to mock `everserver` which I do here, you are correct though, we could run `everserver` and the result would still be correct.